test/file.jl

```diff
@@ -193,7 +193,7 @@ end
             t = i % 2 == 0 ? mktempfile() : mktempdir()
             push!(temps, t)
             @test ispath(t)
-            @test length(TEMP_CLEANUP) == i 
+            @test length(TEMP_CLEANUP) == i 
```

Comment:
?

Reply:
The real question is why that trailing space wasn't caught by the whitespace check...

Reply:
Ah, fun. Our whitespace check only checks for trailing normal spaces.